QSG-pack/src/main/resources/QSG/bin/05/response_unformatted.xml

```diff
@@ -0,0 +1 @@
applicationID>3</ax2140:applicationID><ax2140:applicationName>swift</ax2140:applicationName><ax2140:claimConfig xsi:type="ax2140:ClaimConfig"><ax2140:alwaysSendMappedLocalSubjectId>false</ax2140:alwaysSendMappedLocalSubjectId><ax2140:localClaimDialect>true</ax2140:localClaimDialect><ax2140:roleClaimURI xsi:nil="true"/><ax2140:userClaimURI xsi:nil="true"/></ax2140:claimConfig><ax2140:description>sample service provider</ax2140:description><ax2140:inboundAuthenticationConfig xsi:type="ax2140:InboundAuthenticationConfig"><ax2140:inboundAuthenticationRequestConfigs xsi:type="ax2140:InboundAuthenticationRequestConfig"><ax2140:friendlyName xsi:nil="true"/><ax2140:inboundAuthKey>saml2-web-app-dispatch.com</ax2140:inboundAuthKey><ax2140:inboundAuthType>samlsso</ax2140:inboundAuthType><ax2140:inboundConfigType>standardAPP</ax2140:inboundConfigType><ax2140:properties xsi:type="ax2140:Property"><ax2140:advanced>false</ax2140:advanced><ax2140:confidential>false</ax2140:confidential><ax2140:defaultValue xsi:nil="true"/><ax2140:description xsi:nil="true"/><ax2140:displayName xsi:nil="true"/><ax2140:displayOrder>0</ax2140:displayOrder><ax2140:name>attrConsumServiceIndex</ax2140:name><ax2140:required>false</ax2140:required><ax2140:type xsi:nil="true"/><ax2140:value>1223160755</ax2140:value></ax2140:properties></ax2140:inboundAuthenticationRequestConfigs></ax2140:inboundAuthenticationConfig><ax2140:inboundProvisioningConfig xsi:type="ax2140:InboundProvisioningConfig"><ax2140:dumbMode>false</ax2140:dumbMode><ax2140:provisioningEnabled>false</ax2140:provisioningEnabled><ax2140:provisioningUserStore>PRIMARY</ax2140:provisioningUserStore></ax2140:inboundProvisioningConfig><ax2140:localAndOutBoundAuthenticationConfig xsi:type="ax2140:LocalAndOutboundAuthenticationConfig"><ax2140:alwaysSendBackAuthenticatedListOfIdPs>false</ax2140:alwaysSendBackAuthenticatedListOfIdPs><ax2140:authenticationStepForAttributes xsi:nil="true"/><ax2140:authenticationStepForSubject xsi:nil="true"/><ax2140:authenticationType>default</ax2140:authenticationType><ax2140:enableAuthorization>false</ax2140:enableAuthorization><ax2140:subjectClaimUri>http://wso2.org/claims/fullname</ax2140:subjectClaimUri><ax2140:useTenantDomainInLocalSubjectIdentifier>false</ax2140:useTenantDomainInLocalSubjectIdentifier><ax2140:useUserstoreDomainInLocalSubjectIdentifier>false</ax2140:useUserstoreDomainInLocalSubjectIdentifier></ax2140:localAndOutBoundAuthenticationConfig><ax2140:outboundProvisioningConfig xsi:type="ax2140:OutboundProvisioningConfig"><ax2140:provisionByRoleList xsi:nil="true"/></ax2140:outboundProvisioningConfig><ax2140:owner xsi:type="ax2140:User"><ax2140:tenantDomain>carbon.super</ax2140:tenantDomain><ax2140:userName>cameron</ax2140:userName><ax2140:userStoreDomain>PRIMARY</ax2140:userStoreDomain></ax2140:owner><ax2140:permissionAndRoleConfig xsi:type="ax2140:PermissionsAndRoleConfig"/><ax2140:saasApp>false</ax2140:saasApp></ns:return></ns:getApplicationResponse></soapenv:Body></soapenv:Envelope>
```

Comment:
Add a new  line

Reply:
This is file also contains a SOAP response. Therefore we can't add new lines manually.